cpp/ql/lib/semmle/code/cpp/rangeanalysis/RangeSSA.qll

```diff
@@ -97,8 +97,13 @@ class RangeSsaDefinition extends ControlFlowNodeBase {
    * If this definition is a phi node corresponding to a guard,
    * then return the variable and the guard.
    */
-  predicate isGuardPhi(VariableAccess v, Expr guard, boolean branch) {
-    guard_defn(v, guard, this, branch)
+  predicate isGuardPhi(VariableAccess va, Expr guard, boolean branch) {
+    guard_defn(va, guard, this, branch)
+  }
+
+  /** Whether there exists a frontier node for this phi node */
```

Comment:
```suggestion
  /** Holds if this definition is a _frontier phi node_, meaning it's an ordinary phi node as in non-range SSA. Note that there can be both a guard phi node and a frontier phi node on the same `(def, v)` pair (example: on the statement after `if (x < 2 || ...)`). */
```

I thought this comment should say a bit more about why we added the predicate and why it can be surprising. We'll need a line break in there too.

I can see that the style of the surrounding QLDoc in this file is outdated and inconsistent. For example, we now write "Holds if" instead of "Whether". See [the QLDoc style guide](https://github.com/github/codeql/blob/main/docs/qldoc-style-guide.md#predicates-without-result).